src/WordPress/WordPressClientAPI.php

```diff
@@ -15,10 +15,10 @@ class WordPressClientAPI extends Client
     public function getZoneTag($zone_name)
     {
         $request = new Request('GET', 'zones/', array('name' => $zone_name), array());
-        $response = self::callAPI($request);
+        $response = $this->callAPI($request);
```

Comment:
Don't bother submitting a PR to fix it but in the future you want to use `$this->` to when calling non static members and `self::` when referring to static ones. https://stackoverflow.com/a/151976